imu_filter_madgwick/CMakeLists.txt

```diff
@@ -52,4 +53,17 @@ install(DIRECTORY include/${PROJECT_NAME}/
 
 install(FILES imu_filter_nodelet.xml
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+)
+
+
+
+if(CATKIN_ENABLE_TESTING)
+  catkin_add_gtest(${PROJECT_NAME}-madgwick_test
```

Comment:
That macro is defined in the package `rosunit`. You should add a line `<test_depend>rosunit</test_depend>` to the `package.xml`